# INFERENCIA ESTADISTICA: Tercera practica

## Contrastes de hipótesis: Una población

Esta práctica está relacionada con la práctica anterior (en la que calculamos Intervalos de Confianza y estimaciones). Precisamente, usaremos los mismos comandos y veremos como los tenemos que modificar para realizar contrastes de hipótesis.

In [3]:
# leemos el fichero alturas.txt

Datos <-read.table(file="alturas.txt", header=TRUE, dec=",", sep="\t")

# Para hacer el contraste de varianzas (que haremos en esta práctica), necesitamos instalar y cargar un paquete 
# install.packages("EnvStats")
# library(EnvStats)


### Peso medio de la población 

#### Planteamos el siguiente contraste de hipotesis bilateral 
 $$H_0:\ \mu = 80$$

 $$H_1:\ \mu \neq 80$$

Consideramos un nivel de significación del 1%

In [5]:
t.test(Datos$PESO,alternative='two.sided',mu=80)


	One Sample t-test

data:  Datos$PESO
t = -5.443, df = 170, p-value = 1.805e-07
alternative hypothesis: true mean is not equal to 80
99 percent confidence interval:
 68.97529 76.11243
sample estimates:
mean of x 
 72.54386 


#### Explicación:
El estadístico obtenido es -5.44299, los grados de libertad 170 y el p-valor 1.805397e-07 (es decir, 1.8 por 10 elevado a -7. Con un nivel de significacion del 1%, concluimos que el peso medio de la poblacion no es 80 (ya que el p-valor es menor que 0.01).

#### Observaciones: 
* Si en vez de contrastar el valor 80, se quiere contrastar el valor 60, en vez de mu=80 se debe poner mu=60.
* Para hacer el contraste unilateral a derecha, hay que poner alternative="greater" 
* Para hacer el contraste unilateral a izquierda, hay que poner alternative="less" 
* Observese que, a diferencia de la práctica anterior, no aparece el argumento conf.level en el comando. ¿Por que?

### Contraste de hipotesis de la varianza de las alturas de la poblacion
 $$H_0:\ \sigma^2=16$$

  $$H_1: \ \sigma^2\neq 16$$
  
Consideramos un nivel de significación del 5%

In [4]:
varTest(Datos$ALTURA,alternative  = "two.sided",sigma=16)


	Chi-Squared Test on Variance

data:  Datos$ALTURA
Chi-Squared = 117.75, df = 170, p-value = 0.001614
alternative hypothesis: true variance is not equal to 16
95 percent confidence interval:
  9.057947 13.874455
sample estimates:
variance 
11.08242 


#### Explicacion:
El valor del estadistico obtenido es 117.7507, los grados de libertad de la $\chi^2$ es 170 y el p-valor 0.001614 da. Con un nivel de significación del 5%, se concluye que la varianza de las altueras de la poblacion no es 16 (ya que el p-valor es menor que 0.05). 

#### Observaciones: 
* Si en vez del valor 16, se quiere contrastar el valor 9, en vez de sigma=16 hay que poner sigma=9.
* Para hacer el contraste unilateral a derecha, hay que poner alternative="greater" 
* Para hacer el contraste unilateral a izquierda, hay que poner alternative="less" 
* Observese que, a diferencia de la práctica anterior, no aparece el argumento conf.level en el comando. ¿Por que?


## Contraste de hipotesis de la proporción de los de Alava
$$H_0:\ p\leq 0.15$$

$$H_1:\ p>0.15$$

Consideramos un nivel de significación del 5%.

In [5]:
# contamos cuantos hay de Alava
Datos.alava <- subset(Datos,PROVINCIA==1)
cuantos.alava <- length(Datos.alava$ALTURA)
#cuantos.alava
total <- length(Datos$ALTURA)
#total
prop.test(cuantos.alava,total,p=0.15,alternative="greater")


	1-sample proportions test with continuity correction

data:  cuantos.alava out of total, null probability 0.15
X-squared = 7.5736, df = 1, p-value = 0.002962
alternative hypothesis: true p is greater than 0.15
95 percent confidence interval:
 0.1771247 1.0000000
sample estimates:
        p 
0.2280702 


#### Explicacion:
El valor del estadistico de contraste es 7.573558 y el p-valor 0.002961537. La conclusion es que, con un nivel de significación del 1%, la proporción de alaveses es superior al 0.15 (ya que el p-valor es menor que 0.05).
 
#### Observaciones: 
* Si en vez del valor 0.15, se quiere contrastar el valor 0.4, en vez de p=0.15 hay que poner p=0.4.
* Para hacer el contraste unilateral a izquierda, hay que poner alternative="less" 
* Para hacer el contraste bilateral, hay que poner alternative="two.sided" 
* Observese que, a diferencia de la práctica anterior, no aparece el argumento conf.level en el comando. ¿Por que?